subcommittees/mentoring/procedure-debriefing-session.md

```diff
@@ -0,0 +1,41 @@
+# Procedure - Debriefing Session
+
+This document intends to describe how debriefing sessions
+are organized so that it continue even after members go away.
+
+## Terminology
+
+-   **Debriefing session**: is an online meeting
+    where instructors could share experience about the workshops
+    they taught.
+
+## We Need a Debriefing Session
+
+The Mentoring Subcommittee is responsible to organize
+the debriefing sessions so that
+
+-   each month has at least one debriefing session and
+
+-   instructors are notify with at least one week in advance.
+
```

Comment:
Possibly to consider adding to the operations on the website in the Instructors section that they will be invited and strongly encouraged to attend the debrief. 
Admins, if it's not too much burden, could add a note when they look for instructors "We would also like to invite and strongly encourage you to participate in the debriefing session after the workshop so that you can share your experience and get support, if needed."

I'm suggesting that because I feel that many instructors are not yet aware that these sessions are happening. So letting them know upfront may be nice - they feel like there is a venue to discuss post-workshop stuff.